governance/program/src/state/program_metadata.rs

```diff
@@ -0,0 +1,77 @@
+//! ProgramMetadata Account
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use solana_program::{
+    account_info::AccountInfo, clock::Slot, program_error::ProgramError,
+    program_pack::IsInitialized, pubkey::Pubkey,
+};
+use spl_governance_tools::account::{get_account_data, AccountMaxSize};
+
+use crate::state::enums::GovernanceAccountType;
+
+/// Program metadata account. It stores information about the particular SPL-Governance program instance
+#[repr(C)]
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct ProgramMetadata {
+    /// Governance account type
+    pub account_type: GovernanceAccountType,
+
+    /// The slot when the metadata was captured
+    pub updated_at: Slot,
+
+    /// The version of the program
+    pub version: String,
+
+    /// Reserved
+    pub reserved: [u8; 64],
+}
+
+impl AccountMaxSize for ProgramMetadata {
+    fn get_max_size(&self) -> Option<usize> {
+        Some(85)
```

Comment:
At 85, that gives string data of 8, correct?  1 + 8 + 64 for everything else is 73, then the string is 4 bytes for length, then the data, so 85 - 1 - 8 - 64 - 4 = 8.  That means the longest string would be "XX.YY.ZZ".  If you go fixed length, then you can do a "XXX.YYY.ZZZ", which would definitely cover you forever.

Reply:
Changed the size to support XXX.YYY.ZZZ